app/src/main/java/com/github/houseorganizer/houseorganizer/panels/login/LoginActivity.java

```diff
@@ -88,11 +108,70 @@ protected void onCreate(Bundle savedInstanceState) {
             }
     );
 
+    private void initializeFacebookLogIn() {
+        mCallbackManager = CallbackManager.Factory.create();
+        LoginManager.getInstance().logInWithReadPermissions(this, Arrays.asList("public_profile"));
+        LoginManager.getInstance().registerCallback(mCallbackManager, new FacebookCallback<LoginResult>() {
+            @Override
+            public void onSuccess(LoginResult loginResult) {
+                Log.d(getString(R.string.tag_login_activity), "facebook:onSuccess:" + loginResult);
```

Comment:
we should probably find a convention for when to use logs and when not to, i think until now we havent logged successful actions